ml/ml.cc

```diff
@@ -146,8 +190,128 @@ int test_ml(int argc, char *argv[]) {
     (void) argc;
     (void) argv;
 
+    if(test_ml_anomaly_info_api_sql() == 0) {
+        fprintf(stderr, "PASSED testing ML API SQL query; SQL_SELECT_ANOMALY_RATE_INFO\n");
+    }
+
     ::testing::InitGoogleTest(&argc, argv);
     return RUN_ALL_TESTS();
 }
 
+// --------------------------------------------------------------------------------------------------------------------
```

Comment:
Everything under this line should be in a separate file that will contain code only for tests.

Reply:
Moved to Tests.cc.